apps/discoverbos/widget/HomeProjects.jsx

```diff
@@ -570,14 +570,8 @@ const MobileProjects = () => {
     }
   `;
 
-  const projects = [
-    { asd: "" },
-    { asd: "" },
-    { asd: "" },
-    { asd: "" },
-    { asd: "" },
-    { type: "add" },
-  ];
+  // TODO: Projects should be shown here
+  const projects = JSON.parse(Social.get(`${context.accountId}/thing/project-registry/featured`, "final") || "null") || [];
```

Comment:
@itexpert120 we can use this to display the featured projects. I didn't hook it up to the card

Reply:
Yeah, just need some dummy projects to be indexed, then I can work on integrating them with the project cards. Aftr that this is done.